locust/README.md

```diff
@@ -0,0 +1,23 @@
+# Locust
+
+## Usage
+
+1. Install poetry with dev dependencies.
```

Comment:
Do you mean that the Sketch Map Tool (obviously) has to be set up (with poetry)? In this case, I think one could generally say "set up the tool" as using docker compose, poetry and the other dependencies are installed.

Reply:
Added a comment in 50985c2.